spec/view_helper/applinks_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe MetaTags::ViewHelper, 'displaying Applinks meta tags' do
+  subject { ActionView::Base.new }
+
+  it 'should display meta tags specified with :al' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.